apps/server/src/modules/authentication/controller/dto/password-patter.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { passwordPattern } from './password-pattern';
+
+describe('password patter', () => {
```

Comment:
Neat! Patrick and I discussed the 'tests as documentation' style just yesterday. This is a perfect example :)